src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1796,18 +1796,17 @@ export const ComputeModalBase = ({
                 side: 'bottom'
               }, [renderPersistentDiskType(id)]) : renderPersistentDiskType(id),
```

Comment:
So you shouldn't pass the `id` into this method-- you want to create a unique ID within `renderPersistentDiskType` to link the label "Disk Type" and the Select together.